openstack-details

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+utility_container=`lxc-ls | grep utility`
+
+echo "============================================================="
+echo "openrc"
+echo "============================================================="
+set -x
+lxc-attach -n $utility_container cat /root/openrc | tee ~/.ostrich/openrc
```

Comment:
What does that helper do? Load the openrc? Or is it more clever than that?

I need to read the script and ponder. Maybe I can manage a private backport of it for the purposes of ostrich if its useful. Please hold.